src/cargo_ops/temp_project.rs

```diff
@@ -394,6 +396,13 @@ impl<'tmp> TempProject<'tmp> {
                     }
                 }
                 Value::Table(ref t) => {
+                    let name = match t.get("package") {
+                        Some(&Value::String(ref s)) => s,
+                        // TODO: Probably this should emit an error?
```

Comment:
As far as I understand "package" for a dependency if present can only be a string indicating the name of the real crates.io package. Right now a "package" value which is not a string is silently ignored, but maybe I should turn that into an error too?